res/skins/LateNight/deck.xml

```diff
@@ -307,6 +311,10 @@
             </Children>
           </WidgetGroup>
         </Children>
+	 <Connection>
+	  <ConfigKey persist="true">[Master],show_rate_control</ConfigKey>
+          <BindProperty>visible</BindProperty>
+	 </Connection>
```

Comment:
I can not confirm this on my PC, there it looks good
did you build my branch or just copy the skin folder?
![rate_container](https://user-images.githubusercontent.com/3403218/41938436-620d0da0-7993-11e8-8fbf-f7f407da03d6.png)

Reply:
This PR was branched in April and you didn't merge master since then.
I linked this skin to res folder of a 2.2-master build with qt5.2, from 2018-06-14
2.2.0-alpha-pre (build (no branch) r6513)
If it works for both of us, you should leave the fixed size as it doesn't introduce any regression.

Reply:
>  in April and you didn't merge master since then.

that's not true, I do frequently rebase on upstream master, now I did it again:
![gitk](https://user-images.githubusercontent.com/3403218/41986566-cf160d5e-7a36-11e8-9bd3-f91d7f3e4702.png)

Reply:
> that's not true, I do frequently rebase on upstream master, now I did it again

I didn't see any commit indicating that, so I supposed your version is older.
it doesn't matter anyway as the spinboxes look the same: screwe.
Will fix that separately